src/twisted/names/authority.py

```diff
@@ -14,14 +14,24 @@
 from twisted.names import dns, error, common
 from twisted.internet import defer
 from twisted.python import failure
-from twisted.python.compat import execfile
+from twisted.python.compat import execfile, nativeString, _PY3
+from twisted.python.filepath import FilePath
 
 
-def getSerial(filename = '/tmp/twisted-names.serial'):
-    """Return a monotonically increasing (across program runs) integer.
+
+def getSerial(filename='/tmp/twisted-names.serial'):
```

Comment:
Fixing it is definitely out of scope. And we should probably figure out how to make twistedchecker not complain about stuff that is merely adjacent to your diff rather than part of it...